nums/core/systems/numpy_compute.py

```diff
@@ -237,6 +239,24 @@ def bop(self, op, a1, a2, a1_shape, a2_shape, a1_T, a2_T, axes):
             ufunc = scipy.special.__getattribute__(op)
         return ufunc(a1, a2)
 
+    def bop_reduce(self, op, a1, a2, a1_T, a2_T):
+        if a1_T:
+            a1 = a1.T
+        if a2_T:
+            a2 = a2.T
+
+        reduce_op = np.__getattribute__(op)
+
+        a = np.stack([a1, a2], axis=0)
+        r = reduce_op(a, axis=0, keepdims=False)
+
+        # TODO: Check to see if the others are scalar
```

Comment:
Good practice to leave TODOs. As a convention, please use `TODO (initials): Do this thing.` In this particular instance, could you be more specific about "others"?

Reply:
Good to know about the convention. I already implemented this though, so I'll remove the TODO.